src/hooks/sync-sites/use-sync-pull.ts

```diff
@@ -291,15 +277,14 @@ export function useSyncPull( {
 					pullStatesProgressInfo[ frontendStatus ] || pullStatesProgressInfo.failed;
 				const downloadUrl = hasBackupCompleted ? response.download_url : null;
 
-				if ( hasBackupCompleted && downloadUrl ) {
+				if ( downloadUrl ) {
```

Comment:
What's the context of this removal?

Reply:
The above line has `const downloadUrl = hasBackupCompleted ? response.download_url : null;` - `downloadUrl ` is available only if `hasBackupCompleted ` is `true`. So it's duplication of the condition.